Added a comment for linux users trying the examples (#932)
bevyengine/bevy-website: content/learn/quick-start/getting-started/_index.md

    cargo run --example breakout
    ```

>     If you are using a Linux distro, [install the dependencies](https://github.com/bevyengine/bevy/blob/main/docs/linux_dependencies.md) needed for bevy before trying the examples

Reviewer: ```suggestion
    If you are using Linux, be sure to [install the dependencies](https://github.com/bevyengine/bevy/blob/main/docs/linux_dependencies.md) needed for Bevy before trying the examples.
```
